docs/user_guide/config.md

```diff
@@ -194,152 +194,269 @@ This file adds:
 - [nginx-proxy](https://github.com/nginx-proxy/nginx-proxy) to run NGINX in a container, and automatically generate/refresh NGINX reverse proxy configurations when Neurobagel service containers are started
 - [acme-companion](https://github.com/nginx-proxy/acme-companion) to automatically create and renew SSL certificates for NGINX proxied service containers
 
-??? abstract "Example `docker-compose.yml` with NGINX reverse proxy"
-    To use this file:
 
-    1. If you haven't already, follow the [steps](getting_started.md#the-neurobagel-node-deployment-recipe) to clone and minimally configure the services in the [Neurobagel deployment recipe](https://github.com/neurobagel/recipes)
-    2. Replace the default `docker-compose.yml` in the `recipes` directory with the below file
-    2. Open the file, and edit the values of `VIRTUAL_HOST` and `LETSENCRYPT_HOST` to the domains your proxied services will use (see  :material-plus-circle:)
-        - This assumes you have already registered your domain(s) with a DNS provider and configured the DNS settings to resolve correctly to your host machine
+!!! warning "Community Contribution"
+        
+    **Disclaimer:** This example `docker-compose.yml` is a community-contributed resource and is not officially maintained or supported by the Neurobagel team.
+
+    **Compatibility Note:** This example has been tested with version `v0.4.1` of the [Neurobagel deployment recipes](https://github.com/neurobagel/recipes). Ensure you are using the same or a compatible version for optimal results.
```

Comment:
This is a good warning box, but I think it would be better if we move it under the NGINX/CADDY content tab, so that it is only visible when the CADDY tab is selected. 

To avoid having a warning box inside of the blue `abstract` box, I think we should pull the content tabs out of the `abstract` box - I'll reply below